packages/components/radio/src/radio.css

```diff
@@ -0,0 +1,87 @@
+/* General........................................................ */
+
+:host {
+	& label {
+		display: block;
+		position: relative;
+		padding-left: var(--ts-unit-plus);
+		margin: var(--ts-unit-half);
+		cursor: pointer;
+		font-size: var(--ts-fontsize);
+		-webkit-user-select: none;
+		-moz-user-select: none;
+		-ms-user-select: none;
+		user-select: none;
+
+		&:hover {
+			& span {
+				background-color: var(--ts-color-gray-lighter);
+			}
+		}
+
+		& input {
+			position: absolute;
+			opacity: 0;
+			cursor: pointer;
+			height: 0;
+			width: 0;
+		}
+
+		& span {
+			box-shadow: var(--ts-radio-unselected-boxshadow-border-width) var(--ts-color-gray-light);
+			position: absolute;
+			top: 0;
+			left: 0;
+			height: var(--ts-unit);
+			width: var(--ts-unit);
+			border-radius: 50%;
+
+			&:after {
+				content: '';
+				position: absolute;
+				display: none;
+				top: var(--ts-unit-quarter);
+				left: var(--ts-unit-quarter);
+				width: 8px;
+				height: 8px;
+				border-radius: 50%;
+				background-color: var(--ts-color-white);
+			}
+		}
+	}
+}
+
+:host input:checked ~ span {
+	box-shadow: var(--ts-radio-boxshadow-border-width) var(--ts-color-blue);
+}
+
+/* Disable status .................................................*/
+:host([disabled]) {
+	color: var(--ts-disabled-background-color);
+	cursor: not-allowed;
+
+	&:hover {
+		& span {
+			background-color: transparent;
```

Comment:
It seems that the disabled state of the radio still have the hover effect.

Reply:
The current design of _ts-ui_ doesn't have any hover states, so I removing them here too.

Reply:
Done